lib/WebService/PayPal/PaymentsAdvanced/Response/SecureToken.pm

```diff
@@ -109,6 +177,17 @@ The URL for the PayflowLink web service.  Can be a mocked URL.
 C<Bool> which indicates whether we should pre-fetch the hosted form and do some
 error checking (recommended).
 
+=head2 retry_attempts
+
+The number of HTTP retries to attempt if we encounter an error response. We
+retry only when encountering HTTP 5xx responses.
+
+=head2 retry_callback
+
+The callback to call if we encounter an HTTP 5xx response. We call this prior to
+retrying the request. The callback will have a single parameter, an
+HTTP::Response object.
```

Comment:
From this, it isn't clear what I am supposed to do with the callback or what its purpose is. A sentence about the motivation or with a use case would go a long way in helping the reader understand.